packages/evolution-backend/src/config/projectConfig.ts

```diff
@@ -80,6 +102,19 @@ const projectConfig = Object.assign({}, defaultConfig);
  * @param config The project specific configuration elements
  */
 export const setProjectConfig = (config: Partial<ProjectServerConfig>) => {
+    // Try to set the transition API object from the environment variables
+    if (!config.transitionApi) {
+        const transitionUrl = process.env.TRANSITION_API_URL;
```

Comment:
So we might want to set the URL in the config and the credential in ENV. I don't think this is possible that way ?
But we can keep it that way and say it's an all or nothing way to configure

Reply:
No, all or nothing

Reply:
Added a FIXME to support a mix of both